doc/user-guide/user-guide.rst

```diff
@@ -0,0 +1,598 @@
+  #.  Open dome slit.
+  #.  Once dome is open, open primary mirror cover and vent gates.
+  #.  Enable ``ATAOS`` corrections.
+
+In general, it is advised to make sure all components are in ``ENABLED`` state before running :py:meth:`prepare_for_onsky <lsst.ts.observatory.control.auxtel.ATCS.prepare_for_onsky>`, but the method also accepts a dictionary of ``settings`` and calls :py:meth:`enable <lsst.ts.observatory.control.RemoteGroup.enable>` at the beginning.
+
+.. code:: python
+
+    from lsst.ts.observatory.control.auxtel import ATCS, ATCSUsages
+
+    atcs = ATCS(intended_usage=ATCSUsages.StartUp)
+
+    await atcs.start_task
+
+    # prepare ATCS for flat-field
+    await atcs.prepare_for_onsky()
+
+Following up on what was shown in :ref:`user-guide-generic-csc-group-behavior`, the following is also a valid way of running :py:meth:`prepare_for_onsky <lsst.ts.observatory.control.auxtel.ATCS.prepare_for_onsky>`.
+
+Overriding the settings for a single component (e.g. ATAOS):
+
+.. code:: python
+
+    await atcs.prepare_for_onsky(settings={"ataos": "constant_hex"})
+
+Or Overriding the settings for all components:
+
+.. code:: python
+
+    await atcs.prepare_for_onsky(
+        settings={
+            "ataos": "current",
+            "atmcs": "",
+            "atptg": "",
+            "atpneumatics": "",
+            "athexapod": "current",
+            "atdome": "test.yaml",
+            "atdometrajectory": "",
+        }
+    )
+
+It is important to remember that, if the components are already enabled, they will be left in the ``ENABLED`` state and will not be re-cycled.
+If you need to change the settings for a specific CSC, you will have to send it to ``STANDBY`` state first.
+See :ref:`user-guide-generic-csc-group-behavior` for an example of how to use :py:meth:`set_state <lsst.ts.observatory.control.RemoteGroup.set_state>` to send individual CSCs in the group to ``STANDBY`` state.
+
+All the slew methods discussed in :ref:`user-guide-generic-telescope-control-operations` are available in :py:class:`ATCS <lsst.ts.observatory.control.auxtel.ATCS>`, which can be used with :py:attr:`ATCSUsages.Slew <lsst.ts.observatory.control.auxtel.ATCSUsages.Slew>` to limit resource allocation, e.g.;
+
+.. code:: python
+
+    from lsst.ts.observatory.control.auxtel import ATCS, ATCSUsages
+
+    atcs = ATCS(intended_usage=ATCSUsages.Slew)
+
+    await atcs.start_task
+
+    # Minimum set of parameters.
+    await atcs.slew_icrs(ra="00 42 44.330", dec="+41 16 07.50")
+
+    # Explicitly specify rot_sky and targer_name (both optional).
+    await atcs.slew_icrs(
+              ra="00 42 44.330", dec="+41 16 07.50", rot_sky=0., target_name="M31"
+          )
+
+    # Minimum set of parameters.
+    await atcs.slew_object("M31")
+
+    # Explicitly specify rot_sky (optional).
+    await atcs.slew_object("M31", rot_sky=0.)
+
+For shutting down the observatory :py:meth:`shutdown <lsst.ts.observatory.control.BaseTCS.shutdown>`
+
+.. _user-guide-latiss:
```

Comment:
so these are good but this page is going to get massive! 
we might want to think about how to divide it up

Reply:
yeah...